src/lexer_test.cc

```diff
@@ -33,7 +33,7 @@ TEST(Lexer, ReadEvalStringEscapes) {
   string err;
   EXPECT_TRUE(lexer.ReadVarValue(&eval, &err));
   EXPECT_EQ("", err);
-  EXPECT_EQ("[ $ab c: cde]",
+  EXPECT_EQ("[ ][$][ab c][:][ ][cde]",
```

Comment:
What is happening here? Sorry I don't really understand this part of the code.

Reply:
The AddText optimization to concatenate raw string tokens was removed in favor of keeping an extra token to avoid copying the whole string.  Each escaped character (`$ `, `$$`, `$:`, `$\n`) is parsed and added separately using AddText, so they are now separate tokens in the result.

Reply:
I see, thanks for the explanation :)